admin/views/tabs/tool/import-seo.php

```diff
@@ -47,74 +49,80 @@ function wpseo_import_external_select( $name, $plugins ) {
+				?>
+				<?php
+				do_action( 'wpseo_import_other_plugins' );
+				?>
+				<input type="submit" class="yoast-button yoast-button--primary" name="import_external"
+					value="<?php esc_attr_e( 'Import', 'wordpress-seo' ); ?>"/>
+			</div>
+		</form>
+	</div>
 
-<div class="tab-block">
-	<h3><?php esc_html_e( 'Step 4: Run the configuration wizard', 'wordpress-seo' ); ?></h3>
-	<p>
-		<?php
-		printf(
-			esc_html__( 'You should run the configuration wizard, from the SEO &rarr; General &rarr; Dashboard page, to make sure all the settings for your site are correct.', 'wordpress-seo' ),
-			'<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_dashboard' ) ) . '">',
-			'</a>'
-		);
-		?>
-	</p>
-</div>
+	<div class="tab-block">
+		<h3><?php esc_html_e( 'Step 3: Check your data', 'wordpress-seo' ); ?></h3>
+		<p>
+			<?php esc_html_e( 'Please check your posts and pages and see if the metadata was successfully imported.', 'wordpress-seo' ); ?>
+		</p>
+	</div>
 
-<div class="tab-block">
-	<h3><?php esc_html_e( 'Step 5: Clean up', 'wordpress-seo' ); ?></h3>
-	<p>
-		<?php esc_html_e( 'Once you\'re certain your site is OK, you can clean up. This will remove all the original data.', 'wordpress-seo' ); ?>
-	</p>
-	<form action="<?php echo esc_url( admin_url( 'admin.php?page=wpseo_tools&tool=import-export#top#import-seo' ) ); ?>"
-		method="post" accept-charset="<?php echo esc_attr( get_bloginfo( 'charset' ) ); ?>">
-		<?php
-		wp_nonce_field( 'wpseo-clean-plugins', '_wpnonce', true, true );
-		wpseo_import_external_select( 'clean_external_plugin', $import_check->needs_import );
-		?>
-		<input type="submit" class="button button-primary" name="clean_external"
-			value="<?php esc_attr_e( 'Clean', 'wordpress-seo' ); ?>"/>
-	</form>
+	<div class="tab-block">
+		<h3><?php esc_html_e( 'Step 4: Run the configuration wizard', 'wordpress-seo' ); ?></h3>
+		<p>
+			<?php
+			printf(
+				esc_html__( 'You should run the configuration wizard, from the SEO &rarr; General &rarr; Dashboard page, to make sure all the settings for your site are correct.', 'wordpress-seo' ),
+				'<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_dashboard' ) ) . '">',
+				'</a>'
+			);
+			?>
+		</p>
+	</div>
+
+	<div class="tab-block">
+		<h3><?php esc_html_e( 'Step 5: Clean up', 'wordpress-seo' ); ?></h3>
+		<p>
+			<?php esc_html_e( 'Once you\'re certain your site is OK, you can clean up. This will remove all the original data.', 'wordpress-seo' ); ?>
+		</p>
+		<form action="<?php echo esc_url( admin_url( 'admin.php?page=wpseo_tools&tool=import-export#top#import-seo' ) ); ?>"
+			method="post" accept-charset="<?php echo esc_attr( get_bloginfo( 'charset' ) ); ?>">
+			<div class="yoast-field-group">
+				<?php
+				wp_nonce_field( 'wpseo-clean-plugins', '_wpnonce', true, true );
+				wpseo_import_external_select( 'clean_external_plugin', $import_check->needs_import );
+				?>
+				<input type="submit" class="yoast-button yoast-button--primary" name="clean_external"
```

Comment:
If possible, I prefer a `button type="submit"`

Reply:
Functionality won't work if we replace it with a button